src/components/News/index.js

```diff
@@ -19,7 +19,10 @@ const News = props => {
       <ul className='News'>
         {pages[activePage].map((news, index) => (
           <li className='News__item' key={index}>
-            <strong>{news.intro}:</strong> {news.description}
+            <a href={news.link}>
```

Comment:
When use `<a href="…">`, we actually go through a full page refresh instead of simply updating the main content area. This is why we should use a “router link” instead.

The `Link` component can be imported like this:
```js
import { Link } from 'react-router-dom
```

And then you can use it like this:
```jsx
<Link to={news.link}>
```
Note how the prop is named `to` and not `href` though. ;)